.github/workflows/tagged_release_message.md

```diff
@@ -1,10 +1,16 @@
 This is CBMC Viewer version {}.
 
-To install this release, download the "PIP installation package" below and run
+To install this release using brew, run
+```
+  brew install aws/tap/cbmc-viewer
+```
+
+To install this release using pip, download the "PIP installation package" below and run
```

Comment:
I prefer to say

On MacOS, you can install with brew:  Run
```
brew install aws/tap/cbmc-viewer
```
On all machines, you can install with pip: Download the "PIP installation package" below and run

Reply:
Resolved